src/components/categories/Categories.jsx

```diff
@@ -0,0 +1,37 @@
+import axios from 'axios';
+import React from 'react'
+import { Link } from 'react-router-dom';
+import { useState,useEffect } from 'react';
+import "../categories/categories.css";
+
+ export const Categories = () => {
+     const [categories,setCategories]=useState([]);
+     useEffect(()=>{
+         (async()=>{
+             try {
+               const response= await axios.get("/api/categories")
```

Comment:
const {data} = await axios.get...
In this way you don't need to write response.data...

Reply:
okay